migrations/sql/V2024.06.17.12.00__add_mine_document_bundle_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS mine_document_bundle (
```

Comment:
I just merged a .sql file with the date V2024.07.04.13.26, might need to adjust the date for this sql file I think.

Reply:
Oh, good catch!  Will update the date.